res/res_usbradio.c

```diff
@@ -294,6 +308,30 @@ void ast_radio_put_eeprom(struct usb_dev_handle *handle, unsigned short *buf)
 	buf[EEPROM_USER_CS_ADDR] = (65535 - cs) + 1;
 	write_eeprom(handle, i, buf[EEPROM_USER_CS_ADDR]);
 }
+/*!
+ * \brief See if the passed device matches one of our user defined devices.
+ *
+ * \param dev	usb device
+ * \return 0	does not matches
+ * \return 1	matches
+ */
+static int is_user_device(struct usb_device *dev)
+{
+	struct user_usb_device *device;
+	int match_found = 0;
+	
+	AST_RWLIST_RDLOCK(&user_devices);
+	AST_LIST_TRAVERSE(&user_devices, device, entry) {
+		if(dev->descriptor.idVendor == device->idVendor && 
+			dev->descriptor.idProduct == device->idProduct) {
+			match_found = 1;

```

Comment:
Technically, you don't need the match flag, at the end of traversal device will be NULL, so you can just break, and do `return device ? 1 : 0` or `return device != NULL`, something like that.